### PR TITLE
Add description to pages

### PR DESCRIPTION
**Pull request by pascalberger**
Add description to pages, which will be shown when listing child pages:

image

### CODE REVIEW

File: input/docs/contributing/contribution-guidelines.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 Order: 10
+Description: Guidelines how to contribute to Cake
```
Comment (gep13) [inline input/docs/contributing/contribution-guidelines.md:2]:
```suggestion
Description: Guidelines on how to contribute to Cake
```

File: input/docs/contributing/documentation.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 Order: 20
+Description: Guidelines how to document source code
```
Comment (gep13) [inline input/docs/contributing/documentation.md:2]:
```suggestion
Description: Guidelines on how to document source code
```

File: input/docs/contributing/index.cshtml [hunk 3]
```diff
@@ -1,4 +1,5 @@
 Order: 70
+Description: Information how to contribute to Cake
```
Comment (gep13) [inline input/docs/contributing/index.cshtml:2]:
```suggestion
Description: Information on how to contribute to Cake
```

File: input/docs/extending/index.cshtml [hunk 4]
```diff
@@ -1,5 +1,6 @@
 Order: 60
 Title: Extending Cake
+Description: Documentation how to extend Cake with custom functionality
```
Comment (gep13) [inline input/docs/extending/index.cshtml:3]:
```suggestion
Description: Documentation on how to extend Cake with custom functionality
```

File: input/docs/getting-started/setting-up-a-new-project.md [hunk 5]
```diff
@@ -1,4 +1,5 @@
 Order: 20
+Description: Guide how to get started with Cake
```
Comment (gep13) [inline input/docs/getting-started/setting-up-a-new-project.md:2]:
```suggestion
Description: Guide on how to get started with Cake
```

File: input/docs/integrations/build-systems/index.cshtml [hunk 6]
```diff
@@ -1,4 +1,5 @@
 Order: 90
+Description: Integration with build server
```
Comment (gep13) [inline input/docs/integrations/build-systems/index.cshtml:2]:
```suggestion
Description: Integration with a build server
```

File: input/docs/integrations/index.cshtml [hunk 7]
```diff
@@ -1,4 +1,5 @@
 Order: 50
+Description: Integration with editors, IDEs and build server
```
Comment (gep13) [inline input/docs/integrations/index.cshtml:2]:
```suggestion
Description: Integration with editors, IDEs and build servers
```

File: input/docs/running-builds/index.cshtml [hunk 8]
```diff
@@ -1,4 +1,5 @@
 Order: 30
+Description: Information how to run your Cake builds
```
Comment (gep13) [inline input/docs/running-builds/index.cshtml:2]:
```suggestion
Description: Information on how to run your Cake builds
```

File: input/docs/writing-builds/index.cshtml [hunk 9]
```diff
@@ -1,4 +1,5 @@
 Order: 40
+Description: Information how to write builds
```
Comment (gep13) [inline input/docs/writing-builds/index.cshtml:2]:
```suggestion
Description: Information on how to write builds
```

**Review by gep13 — CHANGES_REQUESTED** [review]

File: input/docs/integrations/build-systems/index.cshtml (hunk 6 above)
Comment (pascalberger) [reply input/docs/integrations/build-systems/index.cshtml:2]: @gep13 I understand your suggestion as "how to integrate with 1 specific build server", while my intention was to say something in the sense of "how to integrate with any build server" (so build server in plural than singular)

Comment (gep13) [reply input/docs/integrations/build-systems/index.cshtml:2]: `Integration with build servers` makes sense to me :+1:

**Review by gep13 — APPROVED** [review]
LGTM

**Pull request merged**